### ISSUE TITLE
Some feature requests

### ISSUE THREAD

**Issue by futurevessel** [smy20011/dreambooth-gui#26]
Is there a chance the following could be added in a future release:

--save_n_steps=500
--save_starting_step=500

--lr_scheduler="polynomial"

Thanks for this great gui version of Dreambooth

**Comment by smy20011** [comment]
Checkpointing is not supported right now and will be supported later.
You can set the `lr_scheduler` in the `Add your custom arguments here.` section.

**Comment by codestuff001** [comment]
Just saw this post where they tested all the learning rate schedulers, their conclusion is marginal difference between them: https://www.reddit.com/r/StableDiffusion/comments/yd56cy/dreambooth_i_compared_all_learning_rate/
